deploy/kubernetes/base/node.yaml

```diff
@@ -17,6 +17,30 @@ spec:
       nodeSelector:
         beta.kubernetes.io/os: linux
       hostNetwork: true
+      affinity:
+        nodeAffinity:
+          requiredDuringSchedulingIgnoredDuringExecution:
+            nodeSelectorTerms:
+            - matchExpressions:
+              - key: "beta.kubernetes.io/os"
+                operator: In
+                values:
+                - linux
```

Comment:
Same above

Reply:
I'll remove the affinity for the `linux`